docs/taskruns.md

```diff
@@ -90,7 +90,7 @@ spec:
         # specifying DOCKER_CONFIG is required to allow kaniko to detect docker credential
         env:
           - name: "DOCKER_CONFIG"
-            value: "/builder/home/.docker/"
+            value: "/tekton/home/.docker/"
```

Comment:
I think it would be nicer to use `$HOME/.docker` here - so we don't promote relying on a specific path

Reply:
Unfortunately, env var values can't be based on other existing env var values. :(

You need a shell involved to actually resolve env vars.

We could do some silly nonsense like replacing the string `$HOME` in env var values before we send it to the Pod, but that feels like a confusing change from K8s behavior that I'd like to avoid.

Reply:
Oh, good point, I see. This is really a pity though.
It might be worth having a built-in variable for that, but I would use Tekton syntax to avoid confusion:

```
       env:
          - name: "DOCKER_CONFIG"
            value: "$(home)/.docker/"
```